### ISSUE TITLE
Publish dependency-check-report.xml to dependency-track fails

### ISSUE THREAD

**Issue by mikezhu8** [DependencyTrack/dependency-track#165]
Hi,

I installed dependency-check plugin (3.2.1)in Jenkins, and deployed dependency-track(md5 is "06cf914f1b363f1e7ca57e30920ec27c") in tomcat 8.5.31.

When "Publish results to Depedency-Track", dependency track always has below error in dependency-track.log.
"2018-06-09 16:19:48,970 [] ERROR [org.owasp.dependencytrack.tasks.ScanUploadProcessingTask] Error while processing scan result
2018-06-09 16:19:48,970 [] ERROR [org.owasp.dependencytrack.tasks.ScanUploadProcessingTask] null"

I tried to tcpdump http put request and response, and then manually run curl command to upload scan, while still see the same issue. Can you please help check?

[scan-upload-cmd.txt.txt](https://github.com/stevespringett/dependency-track/files/2086773/scan-upload-cmd.txt.txt)
[tcpdump-http-put-scan.txt](https://github.com/stevespringett/dependency-track/files/2086774/tcpdump-http-put-scan.txt)

**Comment by stevespringett** [comment]
The Dependency-Check Maven plugin (not sure about others) had a defect from its inception until 3.2.0 which put the format of the GAV between parentheses. In v3.2.1 Dependency-Check fixed this defect by removing the surrounding parentheses and broke all systems that expected that format in the process. In the example you provided, this will fail:

```xml
<identifiers>
    <identifier type="maven" confidence="HIGH">
        <name>commons-io:commons-io:1.4</name>
    </identifier>
</identifiers>
```

This is a duplicate of #156 and has been corrected in master and will be included in the upcoming v3.1 release later this month. For a workaround, use Dependency-Check 3.2.0 or lower.

**Comment by mikezhu8** [comment]
Thanks for Steve's response.
I tried to use 3.2.0 dependency-check CLI, and the XML output includes the same content as 3.2.1. Then I further tried 3.1.2 @http://dl.bintray.com/jeremy-long/owasp/:dependency-check-3.1.2-release.zip, the output XML includes parentheses in GAV, and then call 'api/v1/scan' can upload dependency check result to dependency track successfully, thanks again.

**Comment by stevespringett** [comment]
Closing ticket. Issue has been resolved in 3.1.0
